server/fleet/datastore.go

```diff
@@ -1025,43 +1025,28 @@ type Datastore interface {
 	// WSTEPAssociateCertHash associates a certificate hash with a device.
 	WSTEPAssociateCertHash(ctx context.Context, deviceUUID string, hash string) error
 
-	// MDMWindowsGetEnrolledDevice receives a Windows MDM HW device id and returns the device information.
-	MDMWindowsGetEnrolledDevice(ctx context.Context, mdmDeviceID string) (*MDMWindowsEnrolledDevice, error)
-
 	// MDMWindowsInsertEnrolledDevice inserts a new MDMWindowsEnrolledDevice in the database
 	MDMWindowsInsertEnrolledDevice(ctx context.Context, device *MDMWindowsEnrolledDevice) error
 
 	// MDMWindowsDeleteEnrolledDevice deletes a give MDMWindowsEnrolledDevice entry from the database using the HW device id.
-	MDMWindowsDeleteEnrolledDevice(ctx context.Context, mdmDeviceID string) error
+	MDMWindowsDeleteEnrolledDevice(ctx context.Context, mdmDeviceHWID string) error
 
 	// MDMWindowsGetEnrolledDeviceWithDeviceID receives a Windows MDM device id and returns the device information
 	MDMWindowsGetEnrolledDeviceWithDeviceID(ctx context.Context, mdmDeviceID string) (*MDMWindowsEnrolledDevice, error)
 
 	// MDMWindowsDeleteEnrolledDeviceWithDeviceID deletes a give MDMWindowsEnrolledDevice entry from the database using the device id
 	MDMWindowsDeleteEnrolledDeviceWithDeviceID(ctx context.Context, mdmDeviceID string) error
 
-	// MDMWindowsInsertPendingCommand inserts a command that will be sent to a device
-	MDMWindowsInsertPendingCommand(ctx context.Context, cmd *MDMWindowsPendingCommand) error
-
-	// MDMWindowsInsertPendingCommandForDevices inserts a command that targets multiple devices,
-	// creating one pending command row for each device, with all of the rest of the command
-	// information the same for all devices.
-	MDMWindowsInsertPendingCommandForDevices(ctx context.Context, deviceIDs []string, cmd *MDMWindowsPendingCommand) error
+	// MDMWindowsInsertCommandForDevices inserts a single command that may
```

Comment:
Oops just need to update the method name here.
```suggestion
	// MDMWindowsInsertCommandForHosts inserts a single command that may
```